pkg/lang/golang/queries/find_args.scm

```diff
@@ -0,0 +1,4 @@
+; Finds arguments in an arguments_list
+(argument_list [
```

Comment:
What's the `[` for?

Reply:
so i copied this over from python, must have just been leftover, will remove